src/Avalonia.Controls/ScrollViewer.cs

```diff
@@ -167,6 +167,18 @@ public class ScrollViewer : ContentControl, IScrollable, IScrollAnchorProvider
                 nameof(VerticalScrollBarVisibility),
                 ScrollBarVisibility.Auto);
 
+        /// <summary>
+        /// Defines the <see cref="IsExpandedProperty"/> property.
+        /// </summary>
+        public static readonly DirectProperty<ScrollViewer, bool> IsExpandedProperty =
```

Comment:
Do we need IsExpanded property?
I am not sure, if it is useful anywhere except template.

Reply:
I think this will be used when we detect the scrolling is via touch and set this to false... to prevent expansion? @MarchingCube

Reply:
Currently it is only used in the template, I've added it mostly to keep API similar between `ScrollBar` and `ScrollViewer`. Also might be useful if somebody wants to bind to it. Other option would be a pseudoclass which is basically the same but not public.